vulnerabilities/importers/__init__.py

```diff
@@ -20,9 +20,17 @@
 #  VulnerableCode is a free software code scanning tool from nexB Inc. and others.
 #  Visit https://github.com/nexB/vulnerablecode/ for support and download.
 from vulnerabilities.importers import alpine_linux
+from vulnerabilities.importers import example
 from vulnerabilities.importers import github
 from vulnerabilities.importers import nginx
+from vulnerabilities.importers import npm
 
-IMPORTERS_REGISTRY = [nginx.NginxImporter, alpine_linux.AlpineImporter, github.GitHubAPIImporter]
+IMPORTERS_REGISTRY = [
+    example.ExampleImporter,
+    github.GitHubAPIImporter,
+    nginx.NginxImporter,
+    alpine_linux.AlpineImporter,
+    npm.NpmImporter,
```

Comment:
NPM importer is not migrated yet, so I think we should not have it here

Reply:
oops! My mistake